README.md

```diff
@@ -20,5 +20,5 @@ import cosmokit from 'cosmokit'
 ### Deno
 
 ```ts
-import cosmokit from 'https://cdn.skypack.dev/cosmokit'
-```
\ No newline at end of file
+import cosmokit from 'npm:cosmokit@^1.3.3'
```

Comment:
Is the version number here mandatory? I don't want to have to modify this every update.

Reply:
Not mandatory

Reply:
> Is the version number here mandatory? I don't want to have to modify this every update.

Not mandatory

Reply:
Is it equivalent to `latest` if not provided? 🤔

Reply:
> Is it equivalent to `latest` if not provided? 🤔

Yes, it is.

Reply:
Then why write this `latest` when there is a shorter way to write it?

Reply:
> Then why write this `latest` when there is a shorter way to write it?

I would like to remind that this can be followed by the version.

Reply:
Well I see.